pkg/forklift-api/webhooks/validating-webhook/admitters/secret-admitter.go

```diff
@@ -71,7 +73,22 @@ func (admitter *SecretAdmitter) Admit(ar *admissionv1.AdmissionReview) *admissio
 			if err != nil {
 				log.Info("Connection test failed, yet passing", "status", status, "error", err.Error())
 			} else {
-				log.Info("Test succeeded, passing")
+				// Perform validation for the provided engine CA certificate,
+				// initiate dummy image transfer to make sure the certificate is correct and that the migration will work in a later stage.
+				if isCaOK, err := ovirt.TestDownloadOvfStore(secret); isCaOK {
```

Comment:
yeah, this part needs to be executed specifically for ovirt as you suggested offline